stores/amazon_monitoring.py

```diff
@@ -172,6 +182,9 @@ def __init__(
 
 
 class AmazonMonitor(aiohttp.ClientSession):
+    proxies = dict()
```

Comment:
I'm not an expert at Python, but I find this very awkward. Why is this a dict() and not a list? Below, you are calling a random item from the dict, using a random number as the dictionary key - seems like this should just be a list of dicts.

Reply:
I'm a beginner so I don't know all that well but I've read somewhere that dictionaries are hashed so it is faster to access them but that could be irrelevant in this case since one would be accessing the list via an index value and not iterating over it.

Reply:
Never mind, I think I was getting confused by proxies and self.proxies

Reply:
I do think you're correct about it being awkward - it's an unnecessary layer so I'll get rid of it.